src/_sass/base/_base.scss

```
@@ -265,3 +265,17 @@ blockquote {
     text-align: center;
   }
 }
+
+summary {
+  margin-bottom: 1rem;
+}
+
+ol + img, ul + img, ol + p, ul + p, p + p + img {
+  margin-bottom: 1rem;
+}
+
+.figure-caption {
+  font-size: .875rem;
+  font-style: italic;
+  color: #041E3C;
+}
```

To end the file with a new line:

```suggestion
}

```